packages/migrations/src/batched-migrations/batched-migration-runner.ts

```diff
@@ -177,6 +177,10 @@ export class BatchedMigrationRunner {
       }
 
       if (error) {
+        logger.error(
+          'Error running the batched migration job. To re-run the migration job, open the Admin Panel and click "Retry Failed Jobs".',
+          { error: serializeError(error) },
```

Comment:
Conventionally, we just pass the error object as the second argument (if you have [ast-grep](https://ast-grep.github.io/) installed, try `sg -p 'logger.error($MSG, $ERR)' apps/prairielearn` to see examples):

```suggestion
          error,
```

Reply:
Thanks. I had tried `{ error }` but that caused the content not to show. I have updated it in dbb6a9ecb9c31b8828e608d0b92eb9fa87513c0f.